foundation.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: terra-billing-profile-manager
+  description: |
+    Terra Billing Profile Manager provides an API 
+    to set up and control access to billing within 
+    Terra across cloud platforms.
+  tags:
+    - java
+    - dsp
+    - terra
+    - springboot
+    - broadworkspaces
+  annotations:
+    github.com/project-slug: databiosphere/terra-billing-profile-manager
+spec:
+  type: service
+  lifecycle: production
+  owner: broadworkspaces
+  system: terra
+  dependsOn:
+    - component:sam
+  providesApis:
+    - terra-billing-profile-manager-api
+---
+apiVersion: backstage.io/v1alpha1
+kind: API
+metadata:
+  name: terra-billing-profile-manager-api
+  description: |
+    Placeholder API description...
+  tags:
+    - java
+    - dsp
+    - terra
+    - springboot
+    - broadworkspaces
+  annotations:
+    github.com/project-slug: databiosphere/terra-billing-profile-manager
+spec:
+  type: openapi
+  lifecycle: experimental
```

Comment:
Why is the lifecycle experimental in this one (and production []([here](https://github.com/DataBiosphere/terra-workspace-manager/pull/1522/files#diff-4ac81207abfa7187f9f338eaa84e8800dbce73789e22c14713f4997a3849cd95R47)))?

Reply:
copy paste mistake, thanks!

Reply:
fixed